app/javascript/packs/admin/overview.jsx

```diff
@@ -0,0 +1,91 @@
+import Chart from 'chart.js/auto';
+import { initializeDropdown } from '@utilities/dropdownUtils';
+
+initializeDropdown({
+  triggerElementId: 'timeperiods-trigger',
+  dropdownContentId: 'timeperiods-dropdown',
+});
+
+// Color sets depending on graph trend
+const positiveTrend = {
+  backgroundColor: '#ECFDF5',
+  borderColor: '#10B981',
+};
+const negativeTrend = {
+  backgroundColor: '#FEF2F2',
+  borderColor: '#DC2626',
+};
+
+const charts = document.getElementsByClassName('js-chart');
+
+Array.from(charts).forEach((chart) => {
```

Comment:
Refactor (blocking): Converting the HTMLCollection, e.g. document.getElementsByClassName(...), to an array using `Array.from` is unnecessary. A [HTMLCollection](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCollection) can be iterated over using a [for...of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) statement.

```
for (const chart of charts) {
  ....
}
```

Reply:
Good catch, thanks. It's fixed now.